### PR TITLE
Jarrod

### PR DESCRIPTION
**Pull request by jarrmill**
Created express server, currently writing api endpoint functionality.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    client.query('select * from reviews', (err, res) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(res);
+      }
+    });
+  });
+};
+const selectReviewsByID = function (id) {
```
Comment (partypeoplegames) [inline database/index.js]: pomander boi

File: database/queries.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const idQuery = id => `SELECT * FROM reviews INNER JOIN users ON reviews.user_id=users.id INNER JOIN avatars ON users.avatar_id = avatars.id WHERE reviews.user_id=${id} LIMIT 5;`
```
Comment (NicholasMiron) [inline database/queries.js]: Really long split into several lines. I think you can use \ for a new line

File: server.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  if (!userID){
+    res.statusCode(400);
+    res.send();
+  }
+  selectReviewsByID(userID).then((data) => {
+    console.log('sending: ', data);
+    res.send(data);
+  })
+  .catch(err => {
+    res.send(err);
```
Comment (NicholasMiron) [inline server.js]: Why are you sending your error to the client?

File: server.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  .then(data => {
+    console.log('User Data: ', data);
+    res.send(data);
+  })
+  .catch(err => {
+    console.error('Error in reviews POST ',  err);
+    res.end();
+  })
+})
+app.get('/', (req, res) => res.send('Hello World!'));
```
Comment (NicholasMiron) [inline server.js]: Won't need this later if you send your html

**Pull request closed without merging**